spec/motion-support/core_ext/object/inclusion_spec.rb

```diff
@@ -31,4 +31,18 @@ def include?(obj)
       0.in?(LifeUniverseAndEverything.new).should == false
     end
   end
+
+  describe "presence_in" do
+    it "should return self when included" do
+      "stuff".presence_in(%w( lots of stuff )).should == "stuff"
+    end
+
+    it "should return nil when not included" do
+      "stuff".presence_in(%w( lots of crap )).should == nil
+    end
+
+    it "should raise ArgumentError when arg does not respond to `include?`" do
+      lambda { 1.presence_in(1) }.should.raise ArgumentError
+    end
+  end
```

Comment:
I like what you have done with #in?, but it should really not be part of this pull request.